Source/WebKit/Shared/AuxiliaryProcessExtensions/GPUExtension-Info.plist

```diff
@@ -5,7 +5,7 @@
 	<key>EXAppExtensionAttributes</key>
 	<dict>
 		<key>EXExtensionPointIdentifier</key>
-		<string>com.apple.webkit.gpu.extension</string>
+		<string>com.apple.web-browser-engine.gpu</string>
```

Comment:
Why are these strings changing?

Reply:
I think they were always wrong, and we used to have a post-build script that would fix them up.